unable to follow docker doc umentation

Following the [Docker documentation](https://docs.pycsw.org/en/latest/docker.html#installation):


```
docker run -p 8000:8000 geopython/pycsw
```

the service will return the following error:

```
INFO:pycsw.core.repository:Creating database sqlite:////home/pycsw/pycsw/tests/functionaltests/suites/cite/data/cite.db
INFO:pycsw.core.repository:Creating table spatial_ref_sys
DEBUG:__main__:(sqlite3.OperationalError) table spatial_ref_sys already exists
[SQL: 
CREATE TABLE spatial_ref_sys (
        srid INTEGER NOT NULL, 
        auth_name TEXT, 
        auth_srid INTEGER, 
        srtext TEXT, 
        PRIMARY KEY (srid)
)

]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
DEBUG:pycsw.core.config:Initializing static context
INFO:pycsw.core.repository:creating new engine: sqlite:////home/pycsw/pycsw/tests/functionaltests/suites/cite/data/cite.db
INFO:pycsw.core.repository:binding ORM to existing database
INFO:pycsw.core.repository:setting repository queryables
DEBUG:pycsw.core.repository:Waiting for sqlite:////home/pycsw/pycsw/tests/functionaltests/suites/cite/data/cite.db...
DEBUG:pycsw.core.repository:Database is already up!
DEBUG:__main__:Launching pycsw with gunicorn --bind=0.0.0.0:8000 --access-logfile=- --error-logfile=- --workers=2 --timeout=30 pycsw.wsgi_flask:APP ...
[2025-05-07 08:30:02 +0000] [1] [INFO] Starting gunicorn 23.0.0
[2025-05-07 08:30:02 +0000] [1] [INFO] Listening at: http://0.0.0.0:8000 (1)
[2025-05-07 08:30:02 +0000] [1] [INFO] Using worker: sync
[2025-05-07 08:30:02 +0000] [22] [INFO] Booting worker with pid: 22
[2025-05-07 08:30:02 +0000] [23] [INFO] Booting worker with pid: 23
[2025-05-07 08:30:03 +0000] [22] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/gunicorn/arbiter.py", line 608, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python3.10/site-packages/gunicorn/workers/base.py", line 135, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python3.10/site-packages/gunicorn/workers/base.py", line 147, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python3.10/site-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python3.10/site-packages/gunicorn/app/wsgiapp.py", line 57, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python3.10/site-packages/gunicorn/app/wsgiapp.py", line 47, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python3.10/site-packages/gunicorn/util.py", line 370, in import_app
    mod = importlib.import_module(module)
  File "/usr/local/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/pycsw/pycsw/pycsw/wsgi_flask.py", line 37, in <module>
    from pycsw.ogc.api.records import API
  File "/home/pycsw/pycsw/pycsw/ogc/api/records.py", line 42, in <module>
    from pycsw import __version__
ImportError: cannot import name '__version__' from 'pycsw' (unknown location)
[2025-05-07 08:30:03 +0000] [22] [INFO] Worker exiting (pid: 22)
[2025-05-07 08:30:03 +0000] [1] [ERROR] Worker (pid:22) exited with code 3
[2025-05-07 08:30:03 +0000] [23] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/gunicorn/arbiter.py", line 608, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python3.10/site-packages/gunicorn/workers/base.py", line 135, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python3.10/site-packages/gunicorn/workers/base.py", line 147, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python3.10/site-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python3.10/site-packages/gunicorn/app/wsgiapp.py", line 57, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python3.10/site-packages/gunicorn/app/wsgiapp.py", line 47, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python3.10/site-packages/gunicorn/util.py", line 370, in import_app
    mod = importlib.import_module(module)
  File "/usr/local/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/pycsw/pycsw/pycsw/wsgi_flask.py", line 37, in <module>
    from pycsw.ogc.api.records import API
  File "/home/pycsw/pycsw/pycsw/ogc/api/records.py", line 42, in <module>
    from pycsw import __version__
ImportError: cannot import name '__version__' from 'pycsw' (unknown location)
[2025-05-07 08:30:03 +0000] [23] [INFO] Worker exiting (pid: 23)
[2025-05-07 08:30:03 +0000] [1] [ERROR] Worker (pid:23) exited with code 3
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/gunicorn/arbiter.py", line 208, in run
    self.sleep()
  File "/usr/local/lib/python3.10/site-packages/gunicorn/arbiter.py", line 359, in sleep
    ready = select.select([self.PIPE[0]], [], [], 1.0)
  File "/usr/local/lib/python3.10/site-packages/gunicorn/arbiter.py", line 241, in handle_chld
    self.reap_workers()
  File "/usr/local/lib/python3.10/site-packages/gunicorn/arbiter.py", line 529, in reap_workers
    raise HaltServer(reason, self.WORKER_BOOT_ERROR)
gunicorn.errors.HaltServer: <HaltServer 'Worker failed to boot.' 3>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/gunicorn", line 8, in <module>
    sys.exit(run())
  File "/usr/local/lib/python3.10/site-packages/gunicorn/app/wsgiapp.py", line 66, in run
    WSGIApplication("%(prog)s [OPTIONS] [APP_MODULE]", prog=prog).run()
  File "/usr/local/lib/python3.10/site-packages/gunicorn/app/base.py", line 235, in run
    super().run()
  File "/usr/local/lib/python3.10/site-packages/gunicorn/app/base.py", line 71, in run
    Arbiter(self).run()
  File "/usr/local/lib/python3.10/site-packages/gunicorn/arbiter.py", line 228, in run
    self.halt(reason=inst.reason, exit_status=inst.exit_status)
  File "/usr/local/lib/python3.10/site-packages/gunicorn/arbiter.py", line 341, in halt
    self.stop()
  File "/usr/local/lib/python3.10/site-packages/gunicorn/arbiter.py", line 395, in stop
    time.sleep(0.1)
  File "/usr/local/lib/python3.10/site-packages/gunicorn/arbiter.py", line 241, in handle_chld
    self.reap_workers()
  File "/usr/local/lib/python3.10/site-packages/gunicorn/arbiter.py", line 529, in reap_workers
    raise HaltServer(reason, self.WORKER_BOOT_ERROR)
gunicorn.errors.HaltServer: <HaltServer 'Worker failed to boot.' 3>
```

I have been trying also running the suggested [docker compose](https://github.com/geopython/pycsw/blob/master/docker/compose/docker-compose.yml) config, with same error.

Is there anything we need to configure, or should I checkout a specific branch?

--- a/Dockerfile
+++ b/Dockerfile
@@ -75,7 +75,7 @@ COPY --chown=pycsw . .
 COPY docker/pycsw.yml ${PYCSW_CONFIG}
 COPY docker/entrypoint.py /usr/local/bin/entrypoint.py
 
-RUN pip3 install --editable .
+RUN pip3 install .
 
 WORKDIR /home/pycsw
 

